gigamon/manifest.json

```diff
@@ -0,0 +1,65 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "041cf2fe-f391-4d8b-930c-b700c648c683",
+  "app_id": "gigamon",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": " Deep observability into all application traffic across cloud, virtual, and physical infrastructure",
+    "title": "Gigamon",
+    "media": [
+      {
+        "media_type": "image",
+        "caption": "Gigamon Dashboard for Datadog",
+        "image_url": "images/gd1.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "Gigamon Dashboard for Datadog",
+        "image_url": "images/gd2.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "Gigamon Dashboard for Datadog",
+        "image_url": "images/gd3.png"
+      },
+     {
+        "media_type": "image",
+        "caption": "Gigamon Dashboard for Datadog",
+        "image_url": "images/gigamondashboard.png"
+      }
+
+    ],
+    "classifier_tags": [
+      "Category::AWS",
+      "Category::Azure",
+      "Category::Network",
+      "Category::Security",
+      "Category::Kubernetes",
+      "Category::Containers",
+      "Category::Google Cloud",
+      "Supported OS::Linux",
+      "Supported OS::Windows",
+      "Offering::Integration",
+      "Submitted Data Type::Logs"
+      
+
+    ]
+  },
+  "assets": {
```

Comment:
@emarsha94 can you help provide guidance around adding the logs integration fields here? It's causing a linter validation failure due to the missing `assets.integration` value.

Reply:
@nmuesch do you know why this is failing?

Reply:
It's not that field, but I see other integrations have something like this in their manifest under `assets`: 
```
"integration": {
      "auto_install": <true or false>,
      "source_type_name": "<source name here>"
}
```

Reply:
In order to have a Log Pipeline, the manifest needs an assets->integration section. Here is an example - https://github.com/DataDog/integrations-extras/blob/master/aqua/manifest.json#L30-L46. If your integration doesn't emit any metrics, you can just not include that `metrics` section at all. 

For source_type_id, this is usually generated from `ddev create`, but for this time you can use: `17453472`

Reply:
I can confirm the dashboard was also auto-installed in staging with the pipeline